gpustack/policies/candidate_selectors/gguf_resource_fit_selector.py

```diff
@@ -1097,24 +1176,35 @@ async def find_multi_worker_multi_gpu_candidates(  # noqa: C901
             # Pre filter
             begin_layers = -1
             end_layers = -1
-            for index, item in enumerate(
-                reversed(self._partial_offload_resource_claim.items)
-            ):
-                if not self._model.cpu_offloading:
-                    if index > 0:
-                        break
+            if self._param_gpu_layers:
+                vram_claim_with_with_layers, _ = self._get_claim_with_layers(
+                    self._param_gpu_layers, True
```

Comment:
Why it's always true here?

Reply:
UMA mode requires smaller vram, use this as the minimum required vram.